### PR TITLE
Metrics: rebuild all pull history with cli arg

### LINKED ISSUE #18: metrics: add param to rebuild history
[metrics--add-param-to-rebuild-history](https://github.com/iFixit/pulldasher/compare/metrics--add-param-to-rebuild-history)

### PR DESCRIPTION
**Pull request by davidrans**
Calling `bin/pulldasher rebuild-history` will rebuild the entire
`metrics` DB looping over all pulls, both open and closed.

qa_req 0 (I tested it)

Closes #18

### CODE REVIEW

File: lib/git-manager.js [hunk 1]
```diff
@@ -40,10 +40,11 @@ module.exports = {
     * Returns a promise which resolves to an array of promises. Each promise
     * in the array resolves to GitHub's API response for a Pull Request.
     */
-   getAllPulls: function() {
+   getAllPulls: function(state) {
```
Comment (danielbeardsley) [inline lib/git-manager.js]: I think I'd prefer multiple functions: getAll + getOpen over a strong argument.

**Comment by danielbeardsley** [comment]
CR :+1: with a minor suggestion.

File: lib/git-manager.js (hunk 1 above)
Comment (davidrans) [inline lib/git-manager.js]: okey-doke: a73a677

**Comment by danielbeardsley** [comment]
CR :+1:

**Comment by copperwall** [comment]
Cool! CR :sunglasses:

**Comment by danielbeardsley** [comment]
CR :+1:

**Pull request merged**